### PR TITLE
Add advertisements view. Update page styles. Edit rating in mock data…

### PR DESCRIPTION
**Pull request by gintunas**
… (maximum 5) and add profile to advertisement. Edit profile entity to return int as rating. Add photo to one advertisement. Delete old search results

### CODE REVIEW

**Comment by gintunas** [comment]
Needs further testing and adding functionality and layouts.
First version of working thing is here.

File: src/main/java/com/psk/pyragas/ToolRent/usecases/AddItem.java [hunk 1]
```diff
@@ -18,34 +16,36 @@
...
+//    }
+//
+//    @Transactional
+//    public String createItem() {
+//
+//        itemToCreate.setAdvertisement(this.advertisement);
+//        itemsDAO.persist(itemToCreate);
+//
+//        return "add_item.xhtml?faces-redirect=true&advertisementId=" + this.advertisement.getId();
+//    }
```
Comment (MellowJay) [inline src/main/java/com/psk/pyragas/ToolRent/usecases/AddItem.java]: I guess it's better to leave this code. We might need it for reference when implementing Decorators.

File: src/main/java/com/psk/pyragas/ToolRent/usecases/ExampleItem.java [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+//        orderToCreate.setItems(items);
+//        orderToCreate.setProfile(profile);
+//        ordersDAO.persist(orderToCreate);
+//
+//        // Update item
+//        item.setStatus("Used");
+//        itemsDAO.update(item);
+//
+//        return "index.xhtml?faces-redirect=true";
+//    }
```
Comment (MellowJay) [inline src/main/java/com/psk/pyragas/ToolRent/usecases/ExampleItem.java]: The same idea applies here. Might need this for Decorators.

**Review by MellowJay — CHANGES_REQUESTED** [review]
The code is fine, the only changes that are needed are replacing some english terms with lithuanian ones. The add to cart button is in English. The product info window is called "Product Info" and not something lithuanian like "Produkto Informacija". The rent cost of the item is in dollars. Should probably be in euros.

File: src/main/java/com/psk/pyragas/ToolRent/usecases/AddItem.java (hunk 1 above)
Comment (gintunas) [reply src/main/java/com/psk/pyragas/ToolRent/usecases/AddItem.java]: Yes, I agree. I left it commented out for this reason.

**Review by MellowJay — APPROVED** [review]
Ok, perfect.

**Pull request merged**